src/helpers/citySelectionHelper.ts

```diff
@@ -0,0 +1,21 @@
+import { addToStore, readFromStore, removeFromStore } from '../helpers';
+
+export const SELECTED_CITY = 'selectedCity';
+
+export const loadStoredCity = async () => {
```

Comment:
i suggest `storedSelectedCity` to be more clear. in other cases we have a check and fallback removal if something is not working with the given key, see `voucherHelper`:

```
export const voucherMemberId = async () => {
  let memberId;

  try {
    memberId = await readFromStore(VOUCHER_MEMBER_ID);
  } catch {
    // Token deleted here so that it can be recreated
    removeFromStore(VOUCHER_MEMBER_ID);
  }

  return memberId;
};
```

or as `SELECTED_CITY` is exported here, you could use `await readFromStore(SELECTED_CITY)` directly where needed?

Reply:
Yes. Previously we needed the loadStoredCity function because it updated the state in the helper. But now that we have moved it to `CitySelectionScreen`, we can use it directly with `readFromStore`.